mediaflow_proxy/extractors/voe.py

```
@@ -0,0 +1,59 @@
+from typing import Dict, Any
+from urllib.parse import urljoin
+
+from mediaflow_proxy.extractors.base import BaseExtractor
+
+
+class VoeExtractor(BaseExtractor):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.mediaflow_endpoint = "hls_manifest_proxy"
+
+    async def extract(self, url: str, **kwargs) -> Dict[str, Any]:
```

Ruff (0.12.2)

Unused method argument: `kwargs`

(ARG002)